docs/architecture/psa-thread-safety.md

```diff
@@ -194,13 +201,15 @@ The following functions modify a slot's usage state:
 * `psa_key_derivation_input_key` - reads attr.type
 * `psa_key_agreement_raw_internal` - reads attr.type and attr.bits
 
```

Comment:
Preexisting: broken link at line 175 — `(##destruction of a key in use)` → `(#destruction-of-a-key-in-use)`